src/app/github-landing-page/github-landing-page.component.scss

```diff
@@ -0,0 +1,46 @@
+.cards-container {
+  padding: 2% 0;
+}
+
+.landing-page-card {
+  width: 40%;
+}
+
+.header-container {
+  text-align: center;
+  position: relative;
+}
+
+.close-tab-message {
```

Comment:
Can use the global `text-align-center` class instead:
https://github.com/dockstore/dockstore-ui2/blob/acda6b9d72e5782fc92296ca562dbd5904648cdf/src/styles.scss#L1022-L1024